lib/netbuf/netbuf_read.c

```diff
@@ -10,10 +10,20 @@
 
 #include "netbuf.h"
 
+/*
+ * Set to NULL here; initialized by netbuf_ssl if SSL is being used.  This
+ * allows us to avoid needing to link libssl into binaries which aren't
+ * going to be using SSL.
+ */
+void * (* netbuf_read_ssl_func)(struct network_ssl_ctx *, uint8_t *, size_t,
```

Comment:
../lib/netbuf/netbuf_read.c:18:11: error: no previous extern declaration for
      non-static variable 'netbuf_read_ssl_func'